server.js

```diff
@@ -141,7 +140,7 @@ app.get('/api/v1/pets', (request, response) => {
 
 app.get('/api/v1/pets/:id', (request, response) => {
   const { id } = request.params;
-  const pet = app.locals.pets.find(pet => pet.id === id);
+  const pet = app.locals.pets.find(pet => pet.id === parseIn(id));
```

Comment:
Is this missing a letter t? `parseIn(id)`

Reply:
good catch Paulina!